easybuild/easyconfigs/a/Arb/Arb-2.16.0-iccifort-2018.3.222-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'Arb'
+version = '2.16.0'
+
+homepage = 'http://%(namelower)slib.org'
+
+description = """Arb is a C library for arbitrary-precision interval arithmetic.
+ It has full support for both real and complex numbers. The library is thread-safe,
+ portable, and extensively tested."""
+
+toolchain = {'name': 'iccifort', 'version': '2018.3.222-GCC-7.3.0-2.30'}
+toolchainopts = {'noopt': True}
```

Comment:
@akesandgren This was added by @SethosII based on your suggestion.

But, this corresponds to using `-O0`, which makes the resulting build painfully slow?